packages/api/src/augment/rpc.ts

```diff
@@ -479,5 +479,15 @@ declare module '@polkadot/rpc-core/types.jsonrpc' {
        **/
       version: AugmentedRpc<() => Observable<Text>>;
     };
+    web3: {
```

Comment:
This is auto-generated.

The area to add them is here - https://github.com/polkadot-js/api/blob/master/packages/types/src/interfaces/eth/rpc.ts

Would follow the net example with an `aliasSection` (keeping all the frontier RPCs in a single place). Once `yarn build:interfaces` is run (as on CI), the above is created.